### ISSUE TITLE
always invoke SourcePoint if options set

### ISSUE THREAD

**Issue by dws122** [maithemewp/mai-publisher#20]
It appears SourcePoint is not being called (invoked) if there are no ads on a page.

If SourcePoint is configured it should always run.

see page with zero ads: https://naturesoma.com/refund-policy/

**Comment by JiveDig** [comment]
Yes, right now it's intertwined with the whole ads script. If we want it always, no matter what, it'll take a different approach. Not terribly hard, but not as quick to integrate as the initial version we have now.

**Comment by JiveDig** [comment]
@dws122 Did we already deal with this?
